### ISSUE TITLE
[Feature Request] Hide Scenario Guide without quests

### ISSUE THREAD

**Issue by klewka** [Caraxi/SimpleTweaksPlugin#743]
Hello! I would actually be down to create this but I'd like to make sure it doesn't exist first:

I'd like to hide the scenario guide if both the MSQ and current job quests are completed. That is all.

**Comment by ryankhart** [comment]
I don't know if this setting already in the unmodified game hides the Main Scenario Guide for both MSQ and job quests, but according to the description, it indicates all quests, so maybe this is already a feature in the game already?
![image](https://github.com/Caraxi/SimpleTweaksPlugin/assets/3673962/4240f390-a476-43b5-a276-6cf6b2cd4668)

**Comment by TurtleP** [comment]
I actually ran into this the other day. I wanted to go do one of my mining class quests, but the Main Scenario Guide wasn't showing because of this setting. I had cleared main scenario in its entirety. Turning off the "hide" option fixed it. I even thought it was a tweak I'd enabled.

So basically: "Hide when all quests complete" ignores job/class quests that need to be done and will still hide that UI when MSQ is done. To be honest, I'd rather keep the guide shown at all times. However, I would imagine the solution could be moving the job/class quest to inside the Main Scenario Guide (instead of the text "To Be Continued") and linking it appropriately (as in, shows where to go when you click the UI).

**Comment by klewka** [comment]
Yes, you've both hit the nail on the head. That game option will hide the entire guide (INCLUDING the next job quest) if the MSQ is completed. I just wanted to keep it visible, but that's actually a great suggestion Turtle. I will start taking a look at this once queues settle down.
